src/govuk/components/all.test.js

```diff
@@ -51,3 +55,75 @@ it.each(allComponents)('generate screenshots for Percy visual regression, with J
   await page.goto(baseUrl + '/components/' + component + '/preview', { waitUntil: 'load' })
   await percySnapshot(page, 'js: ' + component)
 })
+
+describe.each(allComponents)('%s', (component) => {
+  describe('examples output valid HTML', () => {
+    const examples = getComponentData(component).examples.map(function (example) {
+      return [example.name, example.data]
+    })
+
+    it.each(examples)('example "%s" outputs valid HTML', async (_, data) => {
+      const $ = render(component, data)
+
+      expect($.html()).toHTMLValidate({
+        rules: {
```

Comment:
This config could all go in to [a `.htmlvalidate.js` in the root of the repo](https://html-validate.org/usage/index.html#configuration), but my thinking is whilst we're only using this in one place we might as well keep it all together?